addons/website_event/views/event_templates_list.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <!-- Index -->
-<template id="index" name="Events">
+<template id="index" name="Events" track="1">
```

Comment:
@pierre-pde Maybe it is time to have a global check on all the controllers and pages and see if everything is tracked as we want to. For instance neither /blog or /shop is tracked right now, while it would be useful?
If you think those need to be enabled, you can just create another task so we don't noise this one.

Reply:
Hi @rdeodoo , as discussed, let's see what can be done. 
We actually have the opportunity to have something great here.